packages/@okta/vuepress-site/code/javascript/okta_sign-in_widget/index.md

```diff
@@ -308,6 +307,56 @@ function error(err) {
 }
 ```
 
+
+## Using with Okta's SDKs
+
+Okta provides a number of SDKs that you might want to use the Sign-In Widget with, including Angular, React, and Vue.
+
+Using the Sign-In Widget with our SDKs that target the web is fairly straightforward.
+
+### Angular
```

Comment:
Should we use a stacked set of tabs for each framework here? If so, I'm not sure how to do it.

Reply:
I think this is okay as-is. The stack selector mechanism was meant more for code samples.

Reply:
This actually looks like a great fit for the stack selector.

The content is specific to and mutually exclusive based on your stack choice – you don't need the Angular content for React, etc – so the stack selector will hide irrelevant content and allow direct links to different stacks.

The stack selector is very general, for any content that depends on a technology choice: Angular versus React, SAML versus OIDC, Azure versus AWS, etc.

Reply:
Is there an example I should use as a guide to make this section use the stack selector?

Reply:
@alexdahl-okta, will we run into the technical issue hit in another guide, where having a stack selector on one page, but no corresponding stack selectors on the other pages, breaks the system? @mraible, I'm happy to do the re-working to make this section into a stack selector, if we decide to go ahead. Can I push commits to your repo?

Reply:
That'd be great @motikagan-okta! Please feel free to push commits to my repo.

Reply:
@mraible, @alexdahl-okta, after experimenting with adding stack selectors to this page, I'm finding it doesn't work,  I think it's because this page is in the `/code` section of `developer.okta.com`, and stack selectors might only supported in the `/docs` section of the site.

Reply:
@bretterer confirmed they're only supported in `/doc`, not here.